.github/workflows/main.yml

```diff
@@ -14,9 +14,9 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - python-version: 3.6
+          - python-version: 3.8
```

Comment:
It usually makes sense to use the latest version of Python possible for static checkers.